benchmark/register-with-gmt-measurement-cluster.py

```diff
@@ -0,0 +1,17 @@
+import requests
```

Comment:
This was in python because I think `requests` was already available any as a dependency. It was python audience, hence demonstrating a very common pythonic way to send HTTP requests.